app/views/hackathons/digest_mailer/organizer_summary.html.erb

```diff
@@ -0,0 +1,10 @@
+
+<p>
+  We've just sent an email about <%= @hackathon.name %> to <%= @count %> hackers. Let us know how things go!
+</p>
+
+<% content_for :signature do %>
+  <p>
+    Best of luck with everything! 🦕 💸
+  </p>
+<% end %>
```

Comment:
```suggestion
<p>
  We've just sent an email about <%= @hackathon.name %> to <%= @count %> hackers.
</p>

<% content_for :signature do %>
  <p>
    Best of luck with everything! 🦕 💸
  </p>
<% end %>
```
Feels kinda repetitive otherwise.